specifyweb/specify/migration_utils/update_schema_config.py

```
@@ -181,21 +303,19 @@ def update_table_schema_config_with_defaults(
             language="en",
         )
 
-        sp_local_container, is_new = Splocalecontainer.objects.get_or_create(
-            name=table_config.name.lower(),
-            discipline_id=discipline_id,
-            schematype=table_config.schema_type,
-            ishidden=False,
-            issystem=table.system,
-            version=0,
+        sp_local_container, is_new = (
```

Ruff (0.15.14)

[warning] Unpacked variable `is_new` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)